diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6fb182e..eff90c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,7 +47,9 @@ target_include_directories(OptickCore PUBLIC "src")
 target_compile_definitions(OptickCore PUBLIC OPTICK_EXPORTS=1)
 if(OPTICK_USE_VULKAN)
        message(STATUS "Optick uses Vulkan")
-       target_include_directories(OptickCore PRIVATE "$(VULKAN_SDK)/Include")
+       include(FindVulkan)
+       target_include_directories(OptickCore PRIVATE ${Vulkan_INCLUDE_DIRS})
+       target_link_libraries(OptickCore PRIVATE ${Vulkan_LIBRARIES})
 else()
        target_compile_definitions(OptickCore PRIVATE OPTICK_ENABLE_GPU_VULKAN=0)
 endif()
 